src/pages/html/tutorials/how-to-create-an-html-button-that-acts-like-a-link/index.md

```diff
@@ -3,13 +3,46 @@ title: How to Create an HTML Button That Acts Like a Link
 ---
 ## How to Create an HTML Button That Acts Like a Link
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/html/tutorials/how-to-create-an-html-button-that-acts-like-a-link/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+This guide will demonstrate "How to Create an HTML Button That Acts Like a Link".
+Programmers and Web Developers use buttons for several different ocassions. 
```

Comment:
Replace "ocassions" with "occasions"

Reply:
Thank you for everything, ill get to the changes soon watching Blacklist atm thank you have a good rest of your night! :)

Reply:
:D

Reply:
Hi James,

I agree with you 100% I looked over the article and the fixes that you said were spot on. Thank you, and if there is anything else you would like to add your more than welcome to.